package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hcaptcha/react-native-hcaptcha",
-  "version": "1.5.1",
+  "version": "1.5.2",
```

Comment:
@emilianodevborn, please also update the version to 1.6.0 as it's technically the new feature

Reply:
@zoryana94 Done!